docs/blog/2020-11-05-zmk-sotf-3.md

```diff
@@ -0,0 +1,156 @@
+
+#### Official USB Product ID
+
+[petejohanson] has gotten an official [USB product ID](https://github.com/openmoko/openmoko-usb-oui/pull/15) assigned to the ZMK Firmware. For anyone looking to uniquely identify a USB device running the ZMK Firmware, you can match on:
+
+- Vendor ID: `0x1d50`
+- Product ID: `0x615e`
+
+We are incredibly grateful that Openmoko Inc., in the wake of discontinuing the openmoko projects, has made this an option for OSS projects.
+
+#### Development: Remote Docker Container Integration
+
+[idan](https://github.com/idan) contributed [VSCode devcontainer integration](https://github.com/zmkfirmware/zmk/pull/209) to make it easier for developers to build and develop ZMK without having to do complicated local toolchain setup and configuration. This also opens up some amazing future flexibility for things like [GitHub Codespaces](https://twitter.com/ZMKFirmware/status/1315760616779505678?s=20).
+
+There's some follow up tweaks necessary for better supporting using this with user config repositories, which will be available soon.
+
+## New Shields
+
+- Reviung41 in [#297](https://github.com/zmkfirmware/zmk/pull/297) - [Nicell]
+- Boardsource 3x4 in [#296](https://github.com/zmkfirmware/zmk/pull/296) - [neegool](https://github.com/neegool)
+- NIBBLE in [#292](https://github.com/zmkfirmware/zmk/pull/292) - [jaygreco](https://github.com/jaygreco)
+- Microdox in [#245](https://github.com/zmkfirmware/zmk/pull/245) - [careyk007](https://github.com/careyk007)
+- MakerDiary M60 in [#233](https://github.com/zmkfirmware/zmk/pull/233) - [megamind4089]
+- TGX4 in [#226](https://github.com/zmkfirmware/zmk/pull/226) - [mubeenkhan94](https://github.com/mubeenkhan94)
+- Querfrency V1 in [#216](https://github.com/zmkfirmware/zmk/pull/216) - [noar-t](https://github.com/noar-t)
+- Cradio in [#224](https://github.com/zmkfirmware/zmk/pull/224) - [davidphilipbarr](https://github.com/davidphilipbarr)
+- Romac+ in [#198](https://github.com/zmkfirmware/zmk/pull/198) - [reizero00](https://github.com/reizero00)
+
+## New Boards
+
+- MakerDiary nrf52840 M.2 Module in [#233](https://github.com/zmkfirmware/zmk/pull/233) - [megamind4089]
+
+## Testing
+
+There has been an _amazing_ amount of testing from various users as we develop new features. In particular, we'd like to call out [tominabox1](https://github.com/tominabox1) who has been tireless in providing detailed and thorough testing of
+changes as they are being developed.
+
+## Coming Soon!
+
+Some items listed in the last coming soon section are still under active development.
+
+- OLED work, including battery and USB/BLE connection status - [petejohanson]
+- One shot mod/layer behaviors - [okke-formsma]
+- A power profiler page for the website, to help users estimate their battery life for a given keyboard - [Nicell]
+- A keymap converter to automatically update keymaps to the new codes and use of `&kp` everywhere - [joelspadin]
+
+## Statistics
+
+Some statistics of interest for ZMK:
```

Comment:
I may not be clear whether these statistics are:
 - since the last SOTF
 - since the start of the project
 - last 30 days (as per website)

Reply:
They are for lifetime unless otherwise noted. Let me add a note about that for those.

Reply:
Updated, added a note about them being lifetime numbers.